index.Rmd

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Unidentified Aerial Phenomenon in America"
-author: Tev'ye davish
-subtitle: 
+author: "Tev'ye davish"
+subtitle: "UFOs"
```

Comment:
also, you can't leave the ":" without a subtitle.  Either add a subtitle or remove that line.